ocs_sample_library_preview/DataView/DataItemResourceType.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class DataItemResourceType(Enum):
+    """
+    enum 1-2  fully inclusive
```

Comment:
this one also has 2 spaces between "2" and "fully"

Reply:
Made this consistent in the enums (one space)